resources/integration_test/tests/query.yaml

```
@@ -0,0 +1,197 @@
+tests:
+  - description: select_star_from_grades_ordered
```

```suggestion
  - description: select_star_ordered
```
See related comment on the following test.